app/customer.py

```diff
@@ -0,0 +1,49 @@
+from app.car import Car
+from app.shop import Shop
+from app.fuel_price import FuelPrice
+
+import datetime
+
+
+class Customer:
+    def __init__(self, data_customer: dict) -> None:
+        self.name = data_customer["name"]
+        self.products = data_customer["product_cart"]
+        self.location = data_customer["location"]
+        self.money = data_customer["money"]

```

Comment:
The customer have car

Reply:
Use self.car = Car(data_customer["car"])

Reply:
After you can remove car argument from functions:
cost_trip
full_cost
shopping